.github/workflows/release.yml

```
@@ -0,0 +1,33 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  ci:
+    uses: ./.github/workflows/ci.yml
```

actionlint (1.7.7)

error while parsing reusable workflow "./.github/workflows/ci.yml": "workflow_call" event trigger is not found in "on:" at line:4, column:3

(workflow-call)